picard/ui/searchdialog/track.py

```diff
@@ -44,36 +44,42 @@
 
 from picard.ui.columns import (
     Column,
+    ColumnAlign,
     Columns,
+    ColumnSortType,
 )
 from picard.ui.searchdialog import (
     Retry,
     SearchDialog,
 )
 
 
+def _track_length_sortkey(track):
+    return track.length
+
+
 class TrackSearchDialog(SearchDialog):
 
     dialog_header_state = 'tracksearchdialog_header_state'
 
     def __init__(self, parent, force_advanced_search=None):
+        self.columns = Columns((
+            Column(N_("Name"), 'title', width=150),
+            Column(N_("Length"), '~length', sort_type=ColumnSortType.SORTKEY, sortkey=_track_length_sortkey, align=ColumnAlign.RIGHT, width=50),
```

Comment:
Instead of `_track_length_sortkey` I think this could use `attrgetter`

Reply:
Done in https://github.com/metabrainz/picard/pull/2658/commits/e127d472da14aff6bfba3d62feb61cbd956c7be2